tests/testthat/test-treatment_effect.R

```diff
@@ -30,33 +30,33 @@ test_that("h_jac_diff work as expected", {
   )
 })
 
-test_that("h_ratio works as expected", {
+test_that("h_risk_ratio works as expected", {
   x <- c(1, 3, 2)
   y <- c(2, 3, 1)
   expect_identical(
-    h_ratio(x, y),
+    h_risk_ratio(x, y),
     c(1 / 2, 1, 2)
   )
   x <- c(1, 2)
   y <- c(2, 4)
   expect_identical(
-    h_ratio(x, y),
+    h_risk_ratio(x, y),
     c(1 / 2, 1 / 2)
   )
 })
 
-test_that("h_jac_ratio work as expected", {
+test_that("h_jac_risk_ratio work as expected", {
```

Comment:
just good to double check that all scales of the contrast are tested in this file?